### ISSUE TITLE
Reduce cache size

### ISSUE THREAD

**Issue by someoneEsle** [wireapp/wire-android#326]
Chances are that if you use Wire on a daily basis like I do (and if you have friends/family scattered around the globe sending pictures 24h/day) at some point the cache used by wire will grow a little too much. The other day I deleted some 2.40Gb of cache -- it felt really good :)
Seriously though, I think there is the need to take care of memory problems, both permanent memory and cache. I hope this will be worked on after the assets shift to version 3. I don't know what the github issues policy is (leave them open until they're fixed, or close them to acknowledge them), so I'll just leave this here. Feel free to close, I just wanted people in the future to have this ticket as reference.

**Comment by someoneEsle** [comment]
Update: after a little over a month, I deleted some more 700+MB of cache, bringing down the total space Wire uses to 0.91GB. Clearly I use Wire fairly often, but adding 700MB/month is a big problem. Consider that in the past month I've also started using timed messages more and more in order to mitigate the memory problems, so the total memory should be much more than just 0.91GB. In a few weeks I will have to clear the data/reinstall, which I won't be particularly happy about since that will mean losing all my conversations on this device. If all my apps took up that much space I'd be reinstalling everything once a month :/
